src/parser/jobs/pld/modules/RoyalAuthority.tsx

```diff
@@ -1,25 +1,31 @@
 import {t} from '@lingui/macro'
 import {Trans} from '@lingui/react'
 import {DataLink} from 'components/ui/DbLink'
-import {Events} from 'event'
 import {Analyser} from 'parser/core/Analyser'
 import {dependency} from 'parser/core/Injectable'
 import Checklist, {TieredRule, TARGET, Requirement} from 'parser/core/modules/Checklist'
 import {Data} from 'parser/core/modules/Data'
 import React from 'react'
 
-const SWORD_OATH_SEVERITY = {
+const ROYAL_AUTHORITY_SEVERITY = {
 	94: TARGET.SUCCESS,
 }
 
 const DIVINE_MIGHT_SEVERITY = {
 	94: TARGET.SUCCESS,
 }
 
-interface SwordOath {
-	initial: number
-	stacks: number
-	used: number
+type Usages = {
+	active: boolean,
+	uses: number,
+	total: number,
+}
+
+type RoyalAuthorityDependentUsages = {
+	divineMight: Usages,
+	atonement: Usages,
+	supplication: Usages,
+	sepulchre: Usages,
 }
 
 export class RoyalAuthority extends Analyser {
```

Comment:
I think there would probably be future benefit to look at whether this module & it's analysis would fit into the Procs module type to be a simpler thing to maintain & be more consistent with output for other classes that have similar functionality, but not something to hold up this PR for.